share/jupyter/voila/templates/base/log.macro.html.j2

```diff
@@ -0,0 +1,35 @@
+{% macro js() %}
+  <!-- voila log js -->
+  <script>
+    const _debug = console.debug;
+    const _info = console.info;
+    const _warn = console.warn;
+    const _error = console.error;
+
+    console.debug = (...args) => {
+        window.top.postMessage({ level: "debug", msg: ["[VOILA]:", ...args] });
+        _debug(...args);
+    };
+
+    console.info = console.info = (...args) => {
```

Comment:
Similar to the leftover comments mentioned in https://github.com/voila-dashboards/voila/pull/671/files#r471981963, we could also remove the superfluous second assignment to `console.info` in a follow-up PR.